html/signup.html

```diff
@@ -0,0 +1,104 @@
+    />
+    <link rel="icon" href="/images/panda_face.svg" type="image/svg" />
+    <link rel="stylesheet" href="../index.css" />
+    <title>판다마켓 회원가입</title>
+  </head>
+  <body class="login-signup-body">
+    <div class="login-signup-container">
+      <header class="login-signup-header">
+        <div class="logo-container">
+          <img src="../images/panda_face.svg" alt="logo" class="logo" />
+          <a href="/"><h2>판다마켓</h2></a>
+        </div>
+      </header>
+
+      <form class="login-signup-form" method="POST">
+        <div class="input-container">
+          <label for="email">이메일</label>
+          <input
+            type="email"
+            id="email"
+            name="email"
+            placeholder="이메일을 입력해주세요"
+          />
+        </div>
+
+        <div class="input-container">
+          <label for="nickname">닉네임</label>
+          <input
+            type="text"
+            id="nickname"
+            name="nickname"
+            placeholder="닉네임을 입력해주세요"
+          />
+        </div>
+
+        <div class="input-container">
+          <label for="password">비밀번호</label>
+          <div class="password-input">
+            <input
+              type="password"
+              id="password"
+              name="password"
+              placeholder="비밀번호를 입력해주세요"
+            />
+            <button class="password-visibility" type="button">
+              <img
+                src="../images/close_eye_grey.svg"
+                alt="closed eye icon"
+                class="close-eye"
+              />
+            </button>
+          </div>
+        </div>
+
+        <div class="input-container">
+          <label for="password-confirm">비밀번호 확인</label>
+          <div class="password-input">
+            <input
+              type="password"
+              id="password-confirm"
+              name="password-confirm"
+              placeholder="비밀번호를 다시 한번 입력해주세요"
+            />
+            <button class="password-visibility" type="button">
+              <img
+                src="../images/close_eye_grey.svg"
+                alt="closed eye icon"
+                class="close-eye"
+              />
+            </button>
+          </div>
+        </div>
+
+        <button type="submit" class="login-signup-button">회원가입</button>
+
+        <div class="oauth-container">
+          <p>간편 로그인하기</p>
+          <div class="oauth-site">
+            <a href="https://www.google.com" class="oauth-button">
+              <img src="../images/google_logo.svg" alt="google login button" />
+            </a>
+            <a href="https://www.kakaocorp.com" class="oauth-button">
+              <img src="../images/kakao_logo.svg" alt="kakao login button" />
+            </a>
+          </div>
+        </div>
+        <div class="switch">
+          이미 회원이신가요?&nbsp;
+          <a href="login.html" class="switch-button">로그인</a>
+        </div>
+      </form>
```

Comment:
💊 제안
현재 form 태그 내에 SNS 로그인 관련 태그들이 포함되어 있습니다. 하지만 form은 제출될 때 필요한 정보만 포함하는 것이 좋습니다.
SNS 로그인 버튼은 실제로 form을 제출하는 과정과 직접적인 연관이 없기 때문에, form 바깥으로 빼는 것이 적절합니다. 이렇게 하면 form의 역할이 명확해지고, 불필요한 데이터가 포함되는 것을 방지할 수 있습니다.

어떤 요소를 form 내부에 포함할지 고민할 때는, 해당 form이 제출될 때 어떤 정보가 서버로 전달되어야 하는지를 기준으로 판단하면 좋습니다!

```suggestion
      </form>
      <div class="oauth-container">
        <p>간편 로그인하기</p>
        <div class="oauth-site">
          <a href="https://www.google.com" class="oauth-button">
            <img src="../images/google_logo.svg" alt="google login button" />
          </a>
          <a href="https://www.kakaocorp.com" class="oauth-button">
            <img src="../images/kakao_logo.svg" alt="kakao login button" />
          </a>
        </div>
      </div>
      <div class="switch">
        이미 회원이신가요?&nbsp;
        <a href="login.html" class="switch-button">로그인</a>
      </div>
```